SeleniumTestXUnit/Tests/Features/Project/Post.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Project Creation
+    As a logged in user, the user should be able to create a project.
+    
+    Background:
+        Given the user is logged in
+
+    @Smoke @Regression @go.to.login.page @delete.project
+    Scenario: Create a new project
+        When the user clicks the New Project button
+            And inputs a new project name
+            And clicks the Add button
+        Then a new project with the chosen name should be displayed in the projects list
+
+    @Negative @go.to.login.page
+    Scenario Outline: Create a new project with an empty name
+        When the user clicks the New Project button
+            And inputs an empty string as the project name
+            And clicks the Add button
+        Then an error message should be displayed indicating that the project name cannot be empty
```

Comment:
Instead of explaining what the error message should display, it is better to show the exact expected message, like this:

Then an error message should be displayed
  And the error message content should be "The project name cannot be empty"

Reply:
In this case, they don't return any error message for the user, how could it be done in this case?